rs/rosetta-api/icrc1/ledger/tests/tests.rs

```diff
@@ -1543,3 +1553,151 @@ mod verify_written_blocks {
         }
     }
 }
+
+mod incompatible_token_type_upgrade {
```

Comment:
General understanding question: this describes the current behaviour but not necessarily the desired behaviour, correct? E.g. it's probably generally not desirable to change the ledger token type at all (even from `u64` to `u256`). Maybe having a TODO with a ticket number here would help clarifying the status. WDYT?

Reply:
Correct. I created a follow-up ticket [FI-1389](https://dfinity.atlassian.net/browse/FI-1389) to consider changing the current behaviour, and added a corresponding `TODO` in the code here.